README.md

```diff
@@ -47,6 +47,17 @@ The units reported here correspond to *Psychopy*'s "normalized units".
 4. Fixation: Blank trials consist of a fixation point presented at the center of the screen for 3s.
     The fixation point is built like in the gaze movement task.
 
+## Resting-state fMRI
+Resting-state fMRI was acquired while the participant watched a naturalistic scene depicting a calm landscape at sunset.
+The movie was captured by a static camera offering the view of the village of Mundaka, located on the northern coast of Spain, as shown in the image below.
+From the original recording of over four hours in length, we extracted 40 minutes, and increased the speed to compress them into exactly 20 minutes.
+As the sun sets, the luminosity gradually decreases.
+The movie contains no sudden or abrupt stimuli, such as faces, aside from occasional passing cars in the village.
```

Comment:
If you really want to be detailed :)

```suggestion
The movie contains no sudden or abrupt stimuli, such as faces, aside from occasional passing cars in the village or one boat in the ocean.
```

Reply:
How did you select which 40-minutes segment to keep? and why 40 minutes (and not 60 minutes compressed 3x e.g)?